x-pack/plugins/alerting/server/task_runner/create_execution_handler.ts

```diff
@@ -71,7 +71,12 @@ export function createExecutionHandler({
     const alertLabel = `${alertType.id}:${alertId}: '${alertName}'`;
 
     for (const action of actions) {
-      if (!actionsPlugin.isActionTypeEnabled(action.actionTypeId)) {
+      if (
+        !actionsPlugin.isActionTypeEnabled(action.actionTypeId) &&
+        !actionsPlugin.preconfiguredActions.find(
+          preconfiguredAction => preconfiguredAction.id === action.id
+        )
+      ) {
```

Comment:
The original error went away but now when I run an alert I get the following:

```
server    log   [15:13:39.655] [error][alerting][alerting][plugins][plugins] Executing Alert "4c476f74-d105-4d2d-8d96-770001095d43" has resulted in Error: action type ".server-log" is not enabled in the Kibana config xpack.actions.enabledActionTypes
```

There seems to be two more places that could benefit from this check:
- https://github.com/elastic/kibana/blob/master/x-pack/plugins/actions/server/create_execute_function.ts#L73
- https://github.com/elastic/kibana/blob/master/x-pack/plugins/actions/server/lib/action_executor.ts#L93

With those, the alert should be able to execute. We could also make a function like `isActionExecutable` in the actions plugin to contain / share this logic.